src/conversions/hsv_to_rgb.rs

```diff
@@ -0,0 +1,60 @@
+// The RGB color model is an additive color model in which red, green, and blue light
+// are added together in various ways to reproduce a broad array of colors. The name
+// of the model comes from the initials of the three additive primary colors, red,
+// green, and blue. Meanwhile, the HSV representation models how colors appear under
+// light. In it, colors are represented using three components: hue, saturation and
+// (brightness-)value. This file provides functions for converting colors from one
+// representation to the other.
+
+pub fn hsv_to_rgb(hue: f32, saturation: f32, value: f32) -> [i32; 3] {
+    if !(0.0..=360.0).contains(&hue)
+        || !(0.0..=1.0).contains(&saturation)
+        || !(0.0..=1.0).contains(&value)
+    {
+        panic!("Input values out of range");
+    }
+
+    let chroma = value * saturation;
+    let hue_section = hue / 60.0;
+    let second_largest_component = chroma * (1.0 - (hue_section % 2.0 - 1.0).abs());
+    let match_value = value - chroma;
+
+    let (red, green, blue) = if (0.0..=1.0).contains(&hue_section) {
+        (chroma, second_largest_component, 0.0)
+    } else if hue_section <= 2.0 {
+        (second_largest_component, chroma, 0.0)
+    } else if hue_section <= 3.0 {
+        (0.0, chroma, second_largest_component)
+    } else if hue_section <= 4.0 {
+        (0.0, second_largest_component, chroma)
+    } else if hue_section <= 5.0 {
+        (second_largest_component, 0.0, chroma)
+    } else {
+        (chroma, 0.0, second_largest_component)
+    };
```

Comment:
What do you think about using [`match`](https://doc.rust-lang.org/reference/expressions/match-expr.html) here?